parse.ts

```diff
@@ -58,23 +65,32 @@ export function parseWithValibot<
 
       const resolveResult = (
         result: SafeParseResult<Schema>,
-      ):
-        | { value: InferOutput<Schema> }
-        | { error: Record<string, string[]> } => {
+      ): { value: InferOutput<Schema> } | { error: ErrorType } => {
         if (result.success) {
           return {
             value: result.output,
           };
         }
 
         return {
-          error: result.issues.reduce<Record<string, string[]>>((result, e) => {
+          error: result.issues.reduce<ErrorType>((result, e) => {
+            if (
+              result === null ||
+              e.message === conformValibotMessage.VALIDATION_UNDEFINED
+            ) {
+              return null;
+            }
+
             const name = formatPaths(
               // @ts-expect-error
               e.path?.map((d) => d.key as string | number) ?? [],
             );
 
-            result[name] = [...(result[name] ?? []), e.message];
+            result[name] =
+              result[name] === null ||
+              e.message === conformValibotMessage.VALIDATION_SKIPPED
+                ? null
```

Comment:
If `error[name]` is null, Conform reuses previous error value: <https://github.com/edmundhung/conform/blob/b0fb91dbab474c7a150d5f56cb71dbb92888779c/packages/conform-dom/form.ts#L931>.

Reply:
Same as the Zod version 👍
 
https://github.com/edmundhung/conform/blob/b0fb91dbab474c7a150d5f56cb71dbb92888779c/packages/conform-zod/parse.ts#L30-L32